### DISCUSSION TITLE
I have added the script in my frontend code but still plausible instance is not showing anything.

### DISCUSSION THREAD

**Discussion by Riteshag05** [plausible/analytics#4609] [Self-Hosted Support]
(1) i cloned this "https://github.com/plausible/community-edition"
(2) changed the BASE_URL, SECRET_KEY_BASE, TOTP_VAULT_KEY
(3) In BASE-URL = http://localhost:8000/, i added this
(4) and for other SECRET AND TOTP , generated their value
(5) after doing this Plausible instance comes up

and also installed the docker

**Comment by ruslandoga** [comment]
Closing as duplicate: https://github.com/plausible/analytics/discussions/4608
